tests/unit/test_icephys.py

```diff
@@ -154,6 +154,13 @@ def test_default(self):
         self.assertEqual(pCS.electrode, electrode_name)
         self.assertEqual(pCS.gain, 1.0)
 
+    def test_gain_optional(self):
+        electrode_name = GetElectrode()
+
+        pCS = PatchClampSeries('test_pCS', list(), 'unit',
+                               electrode_name, timestamps=list())
+        self.assertIsNone(pCS.gain)
+
```

Comment:
Can you add an analogous roundtrip test to make sure read is OK?

Reply:
Added a roundtrip test - let me know if that looks ok!